### DISCUSSION TITLE
Is this map normal?

### DISCUSSION THREAD

**Discussion by SergioRius** [Koenkk/zigbee2mqtt#13608] [General]
All my lights seem to be linked in all combinations between them. Those are three rooms with one being in a 2nd floor.
![imagen](https://user-images.githubusercontent.com/1693727/185679787-8dd40b8b-e9f7-462a-9d36-df66e7c8890e.png)

**Comment by sjorge** [comment]
That looks normal to me, the devices will try to mesh with as much devices as it can keep in it's neighbor table, it doesn't mean it will use all of them, generally it will only forward the message to the shortes route to the destination.  Here is mine:

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/379665/186188853-6e2279ed-b7b8-496e-b8ec-dad88dd1de39.png">

**Comment by comShadowHarvy** [comment]
I have one device in the same room as the coordinator and it links to a router on the other side of the house through five walls then back to the coordinator. So yes it looks normal to me lol
![Screenshot_20220829-215949](https://user-images.githubusercontent.com/2825834/187331467-c3a181f5-a58b-4f07-9497-176a3c5af419.png)

**Comment by SergioRius** [comment]
Lol! Thank you very much, guys.
I guess the last time I saw one of this maps was a zwave one, and that was almost 10 years ago.
